server/modelaggregator.edl

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    trusted {
+        public void enclave_modelaggregator([in, count=length] unsigned char*** encrypted_accumulator,
+            [in, count=length] uint32_t* decrypted_accumulator_lengths,
+            size_t length,
+            [user_check] unsigned char** encrypted_old_params,
```

Comment:
Replace the `user_check` with `in` + `count`. You will need to copy each `char*` in the array into the enclave manually in the function `enclave_modelaggregator`. See the code in Secure XGBoost `ecalls.cpp` -- it contains helper functions that you should be able to copy over directly.